### ISSUE TITLE
Pose estimation data format

### ISSUE THREAD

**Issue by ayazmhmd** [ultralytics/ultralytics#14828]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I am working on a project of teeth and want to detect CEJ bones position. I have the x ray of teeth with the keypoint information as each tooth can have 4 keypoint. the issue i am facing is if a tooth is missing then the 4 keypoint will also be missing in that senerio, Can i train pose estimation model with missing key points

**Comment by Y-T-G** [comment]
Yeah, you can use the `x, y, visibility` format where you set visibility all of them to 0.

**Comment by ayazmhmd** [comment]
> Yeah, you can use the `x, y, visibility` format where you set visibility all of them to 0.

The issue is the data is already annotated and secondly i want to know the keypoint possition matters or not. for example first x,y i have belong to first tooth or it can be of any tooth
x1,y1,x2,y2,x3,y3,x4,y4. always these values be of tooth 1 if it is availble if missing we need to fill it with 0,0 or we can directly ignore it without writing any values

**Comment by Y-T-G** [comment]
Position doesn't matter. You can just not include any label for the missing object.
